templates/422/invalid_protected_area.html

```diff
@@ -0,0 +1,7 @@
+{% extends 'base.html' %}
+{% block content %}
+<h3>Invalid protected area</h3>
+<p>
+  The protected area ID you provided is not supported.
```

Comment:
"not supported" is fine here because we're making arbitrary protected area IDs.

Reply:
I updated the page heading to say "Unsupported protected area" instead of "Invalid protected area" to keep our language consistent, in c1b774a.